gluoncv/model_zoo/mask_rcnn/mask_rcnn.py

```diff
@@ -155,6 +155,30 @@ def reset_class(self, classes, reuse_weights=None):
                     # set data to new conv layers
                     new_params.set_data(new_data)
 
+class MRCNNTargetWrapper(nn.HybridBlock):
+    """Wrapper for the fused Mask RCNN mask targets generator.
+
+    Parameters
+    ----------
+    num_rois : int
+        Number of sampled rois.
+    num_classes : int
+        Number of classes for class-specific targets.
+    mask_size : tuple of int
+        Size of generated masks, for example (14, 14).
+    """
+
+    def __init__(self, num_rois, num_classes, mask_size, **kwargs):
+        super(MRCNNTargetWrapper, self).__init__(**kwargs)
+        self._num_rois = num_rois
+        self._num_classes = num_classes
+        self._mask_size = mask_size[0]
+
+    def hybrid_forward(self, F, roi, gt_mask, matches, cls_targets):
+        return F.mrcnn_target(roi, gt_mask, matches, cls_targets,
```

Comment:
Is it F.contrib.mrcnn_target ?

Reply:
F.mrcnn_target seems to be working.
E.g. if I do `mxnet.nd.mrcnn_target`, it works well

Reply:
Okay, I think I didn't notice that the operator is registered into normal operator namespace. 
To make an operator included in contrib space, it should be https://github.com/apache/incubator-mxnet/pull/16268/files#diff-473b605460620ad0f588dcef298f0d1dR262 `NNVM_REGISTER_OP(_contrib_mrcnn_target)`

Reply:
There's no cpu operator either, so the context is better checked here

Reply:
Can we cahnge the registration to contrib?

Reply:
Ok I will make a MXNet PR for it

Reply:
Updated in https://github.com/apache/incubator-mxnet/pull/16486